File: app/src/main/java/com/neome/api/meta/base/dto/DefnDtoDynamicCondition.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.EnumDefnDynamicOperator;
import com.neome.api.meta.base.Types.MetaIdField;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class DefnDtoDynamicCondition
{
  public MetaIdField lhs;

  public EnumDefnDynamicOperator operator;

  @Nullable
  public FieldDtoArg rhs;
}

File: app/src/main/java/com/neome/api/meta/base/dto/DefnDtoDynamicCondition.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.EnumDefnDynamicOperator
import com.neome.api.meta.base.dto.FieldDtoArg
import com.neome.api.meta.base.Types.MetaIdField

interface DefnDtoDynamicCondition
{
  val lhs: MetaIdField
  val operator: EnumDefnDynamicOperator
  val rhs: FieldDtoArg?
}